### PR TITLE
Add .nvmrc requiring Node LTS version

### PR DESCRIPTION
**Pull request by nsantos16**
## Description
- Added `.nvmrc` indicating node version is LTS

## Notes
Screen Shot 2021-06-14 at 17 45 35
`

### CODE REVIEW

File: template/.nvmrc [hunk 1]
```diff
@@ -0,0 +1 @@
+lts/*
```
Comment (spereztoledo) [inline template/.nvmrc:1]: What do you think about setting this to `14.17` or `14` instead? That way we're locking it to the latest lts, but not allowing it to mean `16` in October, when the next lts is released.

Comment (fsarachu) [reply template/.nvmrc:1]: [Documentation](https://create-react-app.dev/docs/getting-started/#creating-an-app) mentions `Node >= 10` is required
I agree locking ensures consistency, but we will need to remember to update it.
I don't see any major trouble in having it set to `lts/*`. According to CRA, nothing should break, plus we ensure we're not forcing devs to use an outdated version.

Comment (spereztoledo) [reply template/.nvmrc:1]: I don't see any major trouble either, just the unforeseen. I doubt CRA says nothing should break on v16, there's already a problem with node-sass and CRA [reported here](https://github.com/facebook/create-react-app/issues/10944) and [here](https://github.com/facebook/create-react-app/issues/11037). We might also find other libraries we include that have issues with v16.

Those and others that may arise will probably be resolved before v16 is released, but I'd rather be cautious, given that this is the `.nvmrc` that will be copied onto new projects scaffolded with this template, and will live on with them. Best case scenario, it's a simple test and a one line commit to this file once a year when an lts is released. What do you think?

Comment (fsarachu) [reply template/.nvmrc:1]: Oh that's a really great point, it makes total sense to set it to `14` then, so it's locked to that major version 👍

**Pull request merged**